src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx

```diff
@@ -0,0 +1,30 @@
+import { ProjectTable } from './ProjectTable';
+import {
+  projectTableMenuButtonId,
+  projectTableMenuId,
+} from './ProjectTableRow';
+
+describe('Project Table', () => {
+  beforeEach(() => {
+    cy.mount(<ProjectTable />);
+  });
+
+  describe('Action Buttons', () => {
+    it('should open their menu on first click', () => {
+      cy.findAllByTestId(projectTableMenuButtonId).each(el => {
+        const menuBtn = cy.wrap(el);
+        menuBtn.click({ force: true });
+        cy.findByTestId(projectTableMenuId).should('exist');
+      });
+    });
+
+    it('should close their menu on second click', () => {
+      cy.findAllByTestId(projectTableMenuButtonId).each(el => {
+        const menuBtn = cy.wrap(el);
+        menuBtn.click();
+        menuBtn.click();
+        cy.findByTestId(projectTableMenuId).should('not.exist');
+      });
+    });
+  });
+});
```

Comment:
I guess now that you abstracted Action Buttons we should test them there not in places where it is uesed

Reply:
U mean in atoms?

Reply:
yeah

Reply:
I don't know. I'm not sure.

Reply:
what's wrong with doing it?

Reply:
Agree.